yaml/xyz/openbmc_project/Inventory/Item/PCIeSlot.interface.yaml

```diff
@@ -98,6 +104,46 @@ enumerations:
           description: >
               Type of the PCIe slot is unknown
 
+    - name: Status
```

Comment:
From https://github.ibm.com/gmills/pcie-topology

                if device_in_slot.status.state == "Enabled" and \
                        device_in_slot.status.health == "OK":
                    row["Link Status"] = "operational"
                elif device_in_slot.status.state == "Enabled" and \
                        device_in_slot.status.health != "OK":
                    row["Link Status"] = "degraded"
                elif device_in_slot.status.state == "Disabled":
                    row["Link Status"] = "deactivated"
                elif device_in_slot.status.state == "UnavailableOffline":
                    row["Link Status"] = "failed"
                elif device_in_slot.status.state == "StandbyOffline":
                    row["Link Status"] = "inactive"
                    
What do we map Unused, Unknown, and Open to ?

Reply:
What is the difference from Open, inactive and unused? 
Do we need all of them?

Reply:
Well to be honest, I am not sure where these will map. :)
I have just used all the values sent down by Phyp as mentioned in the design document related to this topic.

Reply:
A mapping that includes Open and Unknown: 
```
if device_in_slot.status.state == null or device_in_slot.status.health  == null 
                row["Link Status"] = "Unknown"

if device_in_slot.status.state == "Enabled" and \
                    device_in_slot.status.health == "OK":
                row["Link Status"] = "operational"
            elif device_in_slot.status.state == "Enabled" and \
                    device_in_slot.status.health != "OK":
                row["Link Status"] = "degraded"
            elif device_in_slot.status.state == "Disabled":
                row["Link Status"] = "deactivated"
            elif device_in_slot.status.state == "UnavailableOffline":
                row["Link Status"] = "failed"
            elif device_in_slot.status.state == "StandbyOffline":
                row["Link Status"] = "inactive"
            elif device_in_slot.status.state == "Absent":
                row["Link Status"] = "open"
```